52/lauramariel/pomodoro.py

```diff
@@ -0,0 +1,46 @@
+import time
+from datetime import datetime
+from datetime import timedelta
+
+def timer(timer_type):
+    if timer_type == "pomodoro":
```

Comment:
the only difference here are the number of minutes, so you could turn this into a dictionary: `timer_times = {'pomodoro': 25, 'short_break': 5, 'long_break': 10}`, then you don't have to repeat `endtime = datetime.now() + timedelta(minutes=<int>)` 3 times